src/openapi/streaming/streaming.ts

```diff
@@ -604,10 +604,26 @@ class Streaming extends MicroEmitter<EmittedEvents> {
             return message.Heartbeats;
         }
 
-        if (message.Data && message.Data.length > 0) {
+        if (
+            message.Data &&
+            !Array.isArray(message.Data) &&

```

Comment:
do we use this elsewhere in this library- I'm just aware it needs to be polyfilled for IE11. and in saxotrader, do we polyfill globally or via babel and will it therefore cover this?

I do prefer using Array.isArray if possible...

Reply:
Its used in a couple of places already - https://github.com/SaxoBank/openapi-clientlib-js/blob/aadfd1daaac1f9dc00609ef8c37af83793d75698/src/openapi/streaming/streaming.ts#L560 for example

Reply:
Looks like its supported from IE9 too
https://caniuse.com/?search=Array.isArray